indra/llwindow/llwindow.h

```diff
@@ -63,16 +63,16 @@ class LLWindow : public LLInstanceTracker<LLWindow>
     virtual void show() = 0;
     virtual void hide() = 0;
     virtual void close() = 0;
-    virtual bool getVisible() = 0;
-    virtual bool getMinimized() = 0;
-    virtual bool getMaximized() = 0;
+    virtual bool getVisible() const = 0;
+    virtual bool getMinimized() const = 0;
+    virtual bool getMaximized() const = 0;
     virtual bool maximize() = 0;
     virtual void minimize() = 0;
     virtual void restore() = 0;
-    bool getFullscreen()    { return mFullscreen; };
-    virtual bool getPosition(LLCoordScreen *position) = 0;
-    virtual bool getSize(LLCoordScreen *size) = 0;
-    virtual bool getSize(LLCoordWindow *size) = 0;
+    virtual bool getFullscreen() const { return mFullscreen; };
```

Comment:
Note: `getFullscreen()` was incorrectly left non-virtual.  I fixed it and removed the confuzled comment about it in **llwindowheadless.h**.